main.go

```diff
@@ -124,11 +129,16 @@ func main() {
 		return
 	}
 
+	if *help {
```

Comment:
We can move this to command than it will improve the test coverage. It will enable us to test it as `unit test` returning a nil command meaning that needs to show usage.

Reply:
I'm not sure where you would move this to in there and really why it would help improve things that much. It's probably a bigger change than what I'm looking to do with this PR though.